torchquad/integration/vegas.py

```
@@ -45,7 +53,7 @@ def integrate(
             eps_abs (float, optional): Absolute error to abort at. Defaults to 0.
             max_iterations (int, optional): Maximum number of vegas iterations to perform. Defaults to 20.
             use_warmup (bool, optional): If a warmup should be used to initialize the map. Defaults to True.
-            backend (string, optional): Numerical backend. This argument is ignored if the backend can be inferred from integration_domain. Defaults to "torch".
+            backend (string, optional): Numerical backend. This argument is ignored if the backend can be inferred from integration_domain. "jax" and "tensorflow" are unsupported. Defaults to "torch".
```

What happens if you actually pass backend="jax" or TF though? Can you raise an exception in that case? (e.g. NotImplemented )

---

Yes, I've added an exception for this case now. If "jax" or "tensorflow" was passed before, an exception happened when the first unsupported operation was executed.